src/WPF/WPF.Viewer/Samples/UtilityNetwork/CreateLoadReport/CreateLoadReport.xaml

```diff
@@ -0,0 +1,86 @@
+﻿<UserControl x:Class="ArcGIS.WPF.Samples.CreateLoadReport.CreateLoadReport"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition />
+            <ColumnDefinition />
+        </Grid.ColumnDefinitions>
+        <TextBox x:Name="Phases"
+                 FontSize="20"
+                 TextAlignment="Center" />
+        <Grid Grid.Column="1">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <TextBox Grid.Row="0"
+                     HorizontalAlignment="Stretch"
+                     FontWeight="Bold"
+                     Text="Select a phase type, then click 'Add'"
```

Comment:
Wanted to throw an alternative UI design idea: assign checkboxes or toggle buttons for each phase. This way, no need to list the phases. The instruction could read something like "toggle phases for report"